src/components/grid/GridComponent.vue

```diff
@@ -97,6 +110,28 @@ library.add(faArrowDown, faArrowRight, faArrowsAlt)
 export default Vue.extend({
   name: 'GridComponent',
   components: { FontAwesomeIcon, Loading, GridTitelInfo },
+  computed: {
```

Comment:
Nice, makes working with the grid a lot more clean/clear

Reply:
Thanks!